packages/wrangler/src/init.ts

```diff
@@ -964,6 +964,7 @@ async function getWorkerConfig(
 					if (!(binding as any)?.type) break;
 					configObj.unsafe = {
 						bindings: [...(configObj.unsafe?.bindings ?? []), binding],
+						metadata: configObj.unsafe?.metadata ?? undefined,
```

Comment:
```suggestion
						metadata: configObj.unsafe?.metadata,
```

I think... since metadata is either going to be an object or undefined, right?

Reply:
Hmm, from what I can tell the `configObj.unsafe.metadata` will always be `undefined` since we do not get these values from EWC like we do with unsafe bindings, right?
So I think we can just leave this file untouched, unless we actually want to go get unsafe metadata from EWC (separately from bindings?) and add them into the `configObj` separately?